<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PainelDeRodadas</class>
 <widget class="QDialog" name="PainelDeRodadas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="twgIntegrantesRodada">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>110</y>
     <width>1251</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>70</y>
     <width>1251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIndicativoComentarios">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>70</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>Indicativo Selecionado ::</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstatisticaFila">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>70</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>Fila ::</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstatisticaQap">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>70</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>QAP ::</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstatisticaQrx">
   <property name="geometry">
    <rect>
     <x>929</x>
     <y>70</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>QRX ::</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstatisticaQrt">
   <property name="geometry">
    <rect>
     <x>1169</x>
     <y>70</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>QRT ::</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalParticipantes">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>70</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>Amigos Radioamadores ::</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/sair.png</normaloff>:/sair.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAtualizarPainelRodada">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/recarregar.png</normaloff>:/recarregar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQrt">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QRT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnObs">
   <property name="geometry">
    <rect>
     <x>487</x>
     <y>10</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OBS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnColarIndicativo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/colar.png</normaloff>:/colar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditarRA">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/radioamadores.png</normaloff>:/radioamadores.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQap">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QAP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvarComentariosDaRodada">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/salvar.png</normaloff>:/salvar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/PessoaRemover.png</normaloff>:/PessoaRemover.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtIndicativo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>24</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnParticiparRodada">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/participarRodada.png</normaloff>:/participarRodada.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQrx">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QRX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFim">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/finalizar.png</normaloff>:/finalizar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>470</y>
     <width>401</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Rodada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtRodada">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>524</y>
     <width>271</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Coordenador</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCoordenador">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>524</y>
     <width>61</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ID Rodada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtFkRodada">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>524</y>
     <width>51</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Edição</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEdicao">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>577</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Comentário da Rodada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtComentarios"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>470</y>
     <width>301</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Inserir Comentário para o Radioamador</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtComentariosRA"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>526</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Comentários atuais</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="twgComentariosRA"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>470</y>
     <width>401</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Log de Participação</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="twgLogParticipacao"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnRemoverComentario">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>470</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInserirComentario">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
